dss_datamover/systemtests/test_datamover.py

```diff
@@ -91,3 +93,19 @@ def test_delete_operation(self, get_master, reset_master_obj, setup_data_dir):
         get_master.start()
         process_del_operation(get_master)
         assert get_master.testcase_passed.value
+
+    def test_put_large_file(self, get_master, reset_master_obj, setup_data_dir, setup_large_file, get_pytest_configs):
+        print("Testing PUT operation with large file")
+        get_master.operation = "PUT"
+        get_master.fs_config['nfs'] = {'127.0.0.1': get_pytest_configs['large_file_path']}
```

Comment:
can we use localhost instead of ip addr?

Reply:
Yes of course. Changed.